gulpfile.js

```diff
@@ -75,14 +79,32 @@ gulp.task('browserify', function() {
     .pipe(gulp.dest(paths.distPublic));
 });
 
-gulp.task('build', ['build-queries', 'copy-js', 'transform-jsx', 'browserify']);
+gulp.task('less', function () {
+  var combined = combiner.obj([
+    gulp.src(paths.lessBundle),
+    less({
+      paths: [
+        'node_modules/bootstrap/less'
+      ]
+    }),
+    gulp.dest(paths.distPublic + '/')
```

Comment:
What about `paths.less`? Is there a way to use that here, since it's the thing you're watching?